src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -459,12 +459,10 @@ internal bool CheckValueKind(SyntaxNode node, BoundExpression expr, BindValueKin
                     // Note: RValueOnly is checked at the beginning of this method. Since we are here we need more than readable.
                     //"this" is readonly in members of readonly structs, unless we are in a constructor.
```

Comment:
>"this" is readonly in members of readonly structs, unless we are in a constructor [](start = 22, length = 81)

Does this comment need updating?

Reply:
Yeah, I should update it to describe the the new readonly members scenario.